src/server/status/server_status.js

```diff
@@ -16,6 +17,7 @@ module.exports = class ServerStatus {
   }
 
   createForPlugin(plugin) {
+    plugin.version = plugin.version || version;
```

Comment:
I wonder if this is too loose? I like what it's enabling, but I wonder if the behavior should be explicit?

Instead of removing `version` from every built-in plugin, just mark it with `"version" : "kibana"`, or something similar, and use that to inherit Kibana's version. That way it at least implies to plugin authors that they should provide their own version number and not just make version irrelevant for every plugin ("oh, it always matches Kibana", which shouldn't be true for things that are compiled + released at the same time).